packages/yoshi-helpers/src/utils.ts

```diff
@@ -226,7 +226,7 @@ export const getProjectCDNBasePath = (useUnversionedBaseUrl: boolean) => {
     // Webpack's `publicUrl` to use the first option.
     artifactPath = 'dist';
   } else {
-    artifactPath = getProjectArtifactVersion();
+    artifactPath = getProjectArtifactVersion() || '';
```

Comment:
![future-is-now](https://media.giphy.com/media/jp8lWlBjGahPFAljBa/giphy.gif)

```suggestion
    artifactPath = getProjectArtifactVersion() ?? '';
```